cli/tsc/00_typescript.js

```diff
@@ -125170,14 +125170,6 @@ function changeCompilerHostLikeToUseCache(host, toPath3, getSourceFile) {
     }
     return sourceFile;
   } : void 0;
-  host.fileExists = (fileName) => {
-    const key = toPath3(fileName);
-    const value = fileExistsCache.get(key);
-    if (value !== void 0) return value;
-    const newValue = originalFileExists.call(host, fileName);
-    fileExistsCache.set(key, !!newValue);
-    return newValue;
-  };
```

Comment:
Removes this cache because we always return `false` (which is questionable and something we should look into in the future, but no need to have a cache for a constant)